test/controllers/EditSetControllerSpec.scala

```diff
@@ -651,6 +651,88 @@ class EditSetControllerSpec extends BaseSpec {
 
       }
 
+      "Covering dates, normalizing direction names" in {
+
+        val page = orderingRequest("covering-dates", "Descending")
+        status(page) mustBe OK
+
+        assertPageAsExpected(
+          asDocument(page),
+          ExpectedEditSetPage(
+            title = "Browse Edit Set (Page 1 of 2)",
+            caption = "Showing 1 - 10 of 12 records",
+            header = "Edit set: COAL 80 Sample",
+            ccrTableHeader = ExpectedTableHeader("ccr", "ascending"),
+            scopeAndContentTableHeader = ExpectedTableHeader("scope-and-content", "ascending"),
+            coveringDateTableHeader = ExpectedTableHeader("covering-dates", "ascending"),
```

Comment:
Likely out of scope, but I just noticed that we cannot tell which of the headers is the "active" one (that is, ordered). However, from the order of the records, it's clear.

Reply:
Hm, we could add the CSS selectors for the `aria-sort` attribute here.